agimus_controller/agimus_controller/ocp_base_croco.py

```diff
@@ -27,7 +27,7 @@ def __init__(
         # Setting the robot model
         self._robot_models = robot_models
         self._collision_model = self._robot_models.collision_model
-        self._armature = self._robot_models._params.armature
+        self._armature = self._robot_models.params.armature
```

Comment:
See:
https://github.com/agimus-project/agimus_controller/pull/170/files#diff-96ef9ae79ce59c979f891751a1aeb04245efdca540e6072b3e4f4f0dc0e94a42R246
The armature is available directly from the `robot_models` object.
```suggestion
        self._armature = self._robot_models.armature
```

Reply:
As a private attribute...